practice_app/apps/user/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+class User(models.Model):
+    username = models.CharField(max_length=50, unique = True)
+    password = models.CharField(max_length=50)
```

Comment:
Maybe it would be better to make this field as a password type. You can check [this link](https://stackoverflow.com/a/17531053) for your convenience.

Reply:
Thanks for the link, I will work on that.